students/slawinski_amadeusz/lesson_04_unit_testing/test_distance.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+
+import math
+
+
+def distance(x1, y1, x2, y2):
+    return math.sqrt(math.pow(float(x2) - float(x1), 2) +
+                     math.pow(float(y2) - float(y1), 2))
+
+
+def test_TypeError_with_None():
```

Comment:
użyj pytes: `import pytest`, dzieki czemu sprowadzi się to do:
 `....with pytest.raises(TypeError):`
`..........distance(None,0, 0, 0)`
Po za tym, gdy wywołasz `distance(None)` dostaniesz TypeaError nie dlatego, że użyłeś `None` tylko dlatego, ponieważ podałeś jeden argument zamiast 3 (`distance(3)` tez zwróci `TyoeError`)